lib/Dialect/FIRRTL/Transforms/ExpandWhens.cpp

```diff
@@ -94,6 +107,37 @@ class ExpandWhensVisitor : public FIRRTLVisitor<ExpandWhensVisitor> {
         });
   }
 
+  /// For every leaf field in the sink, record that it exists and should be
+  /// initialized.
+  void declareSinks(Value value, Flow flow) {
+    auto type = value.getType();
+
+    // A sink location which will be modified as we traverse any bundle type.
+    FieldRef sink(value);
+
+    // Recurse through a bundle and declare each leaf sink node.
+    std::function<void(Type, Flow)> declare = [&](Type type, Flow flow) {
+      // If the field is flipped, flip the flow and use the real type.
+      if (auto flip = type.dyn_cast<FlipType>()) {
+        type = flip.getElementType();
+        flow = swapFlow(flow);
+      }
+      // If this is a bundle type, recurse to each of the fields.
+      if (auto bundleType = type.dyn_cast<BundleType>()) {
+        for (auto &it : llvm::enumerate(bundleType.getElements())) {
```

Comment:
you don't use the index from enumerate, so you don't need it.

Reply:
Fixed!